src/ert/scheduler/__init__.py

```diff
@@ -22,50 +24,39 @@ def create_driver(config: QueueConfig) -> Driver:
     if config.queue_system == QueueSystem.LOCAL:
         return LocalDriver()
     elif config.queue_system == QueueSystem.TORQUE:
-        queue_options = dict(config.queue_options.get(QueueSystem.TORQUE, []))
-        num_nodes: Optional[str] = queue_options.get("NUM_NODES")
-        num_cpus_per_node: Optional[str] = queue_options.get("NUM_CPUS_PER_NODE")
-        return OpenPBSDriver(
-            qsub_cmd=queue_options.get("QSUB_CMD"),
-            qstat_cmd=queue_options.get("QSTAT_CMD"),
-            qdel_cmd=queue_options.get("QDEL_CMD"),
-            queue_name=queue_options.get("QUEUE"),
-            keep_qsub_output=queue_options.get("KEEP_QSUB_OUTPUT", "0"),
-            memory_per_job=queue_options.get("MEMORY_PER_JOB"),
-            num_nodes=int(num_nodes) if num_nodes else None,
-            num_cpus_per_node=int(num_cpus_per_node) if num_cpus_per_node else None,
-            cluster_label=queue_options.get("CLUSTER_LABEL"),
-            job_prefix=queue_options.get("JOB_PREFIX"),
-            project_code=queue_options.get("PROJECT_CODE"),
-        )
+        queue_dict = asdict(config.queue_options)
+        queue_dict["queue_name"] = queue_dict.pop("queue")
+        queue_dict.pop("max_running")
+        queue_dict.pop("submit_sleep")
+        return OpenPBSDriver(**queue_dict)
```

Comment:
Should it be a helper function in the TorqueOptions class to provide this argument dictionary for all drivers?

Reply:
Now included as a bonus commit (to be squashed)